### ISSUE TITLE
SegFormer baseline

### ISSUE THREAD

**Issue by linyuhongg** [GuoleiSun/VSS-MRCFA#4]
Hi, is it possible to get the code base for reproducing the baseline (SegFormer) results ?

**Comment by GuoleiSun** [comment]
Hello, thanks for you interest. Sorry for the late reply. For segformer baseline, you can simply run the configs of CFFM, but you need to change the loss in [this place](https://github.com/GuoleiSun/VSS-MRCFA/blob/d2cd5ecc1ef31e8d192aa28a34e26275802256be/mmseg/models/decode_heads/decode_head.py#L543). From line 543 to line 552, change them to
```
loss['loss_seg'] = self.loss_decode(
            seg_logit_ori,
            seg_label_ori,
            weight=seg_weight,
            ignore_index=self.ignore_index)
```
This means to comment out the loss for our video module, while only keeping the the original losses (for each separate frame). In this way, all the training settings for the baselines are kept the same as CFFM. Thus the comparisons are fair. Please let me know you have more questions.
